README.md

```diff
@@ -1 +1,6 @@
-# hiprfish-imaging
\ No newline at end of file
+# HiPR-FISH Spatial Mapping of Cheese Rind Microbial Communities 
+# :cheese: :microscope: :world_map:
+
+[HiPR-FISH spatial imaging](https://doi.org/10.1038/s41586-020-2983-4) was used to look at the distribution of microbes within five distinct microbial communities growing on the surface of aged washed rind cheeses (link to pub doi). HiPR-FISH was performed by [Kanvas Biosciences](https://www.kanvasbio.com). Imaging data and spatial correlation data is available at (zenodo link). 
```

Comment:
do you have the DOIs yet for the pub and for zenodo? if so we can add them now, and if not it can be a last minute change!

Reply:
not yet! but I will add a row in Monday to make sure I do that before I create a release